### DISCUSSION TITLE
i want to know hosp.labelevenets include icu label or not?

### DISCUSSION THREAD

**Discussion by ggsimidaa168** [MIT-LCP/mimic-code#1661] [MIMIC-IV] [answered]
i want to know hosp.labelevenets include icu label or not? 
i'll be grateful for any helpful answer ！

**Comment by heisenbug-1** [comment, marked as answer]
Hi!
As far as I know, there's no icu stay_id in the hosp.labevents table, but you can join hosp.labevents with icu.icustays on subject_id and hadm_id, and then filter by time(measurement should be within icu intime and outtime), since hadm_id can be mapped to multiple icu stay_id.

You can also use icu.chartevents. It contains less data than hosp.labevents, but it has icu labels.

**Reply by ggsimidaa168** [reply]
thanks for your help！
